contracts/Configuration/Time.sol

```diff
@@ -19,7 +19,11 @@ contract Time {
 
     uint64 constant public PAYOUT_PERIOD_DURATION = 28 days;
     uint64 constant public PAYOUT_TO_STORAGE_PERIOD_MULTIPLIER = 13;
-    uint64 constant public STORAGE_PERIOD_DURATION = 364 days;
+    uint64 public STORAGE_PERIOD_DURATION;
+
+    constructor() public {
+        STORAGE_PERIOD_DURATION = PAYOUT_PERIOD_DURATION.mul(PAYOUT_TO_STORAGE_PERIOD_MULTIPLIER).castTo64();
+    }
```

Comment:
These are const values so we don't really need sefemath here. Also this can be computed in compile-time
```
    uint64 constant public STORAGE_PERIOD_DURATION = PAYOUT_PERIOD_DURATION * PAYOUT_TO_STORAGE_PERIOD_MULTIPLIER;
```

Reply:
++